docs/source/pages/SJ2088POE.rst

```diff
@@ -50,7 +50,7 @@ Key features
 Minimal and maximal perceiving distances of the camera
 ******************************************************
 
-Minimal depth perceiving distance of the camera depends on mono camera FOV, resolution, baseline and stereo depth mode, more info is available on the `Stereo Depth documentation <https://docs.luxonis.com/projects/api/en/latest/components/nodes/stereo_depth/>`__ and `here <https://docs.luxonis.com/en/latest/pages/faq/#id1>`__.
+Minimal depth perceiving distance of the camera depends on mono camera FOV, resolution, baseline and stereo depth mode, more info is available on the `Stereo Depth documentation <https://docs.luxonis.com/projects/api/en/latest/components/nodes/stereo_depth/#min-stereo-depth-distance>`__ and `here <https://docs.luxonis.com/en/latest/pages/faq/#id1>`__.
```

Comment:
is there any valuable info in faq that's not in the stereoDepth docs? If so, it should be included in stereoDepth, otherwise we can remove the link to faq.

Reply:
This was before we updated StereoDepth docs, removed the link to FAQ in every instance.